Parser/convertSBEcnvVar.m

```diff
@@ -122,12 +122,23 @@
     case 'flC'
       name = 'CPHL';
       comment = 'Artificial chlorophyll data computed from bio-optical sensor raw counts measurements. Originally expressed in ug/l, 1l = 0.001m3 was assumed.';
-      
+
+    %fluorescence (ug/l)
+    %Aquatracka SBE19plus cdom sensor as chpl implemented on #550
+    case 'flCUVA'
+      name = 'CPHL';
+      comment = 'Artificial chlorophyll data computed from bio-optical sensor raw counts measurements (Chelsea UV Aquatracka). Originally expressed in ug/l, 1l = 0.001m3 was assumed.';
+
+    % PAR/Irradiance, Biospherical/Licor
+    %par/sat/log - light sensor in SBE19plus implemented on #550
+    case {'par0x2Fsat0x2Flog', 'par/sat/log'}
+      name = 'PAR';
+      comment = 'PAR/Logarithmic/ Satlantic, assumed in umol photons/m2/s';
```

Comment:
Please remove mention of unit there. You can put this information in commented code if you want.

Reply:
amended again